R/get_predicted.R

```diff
@@ -145,10 +146,11 @@ get_predicted.data.frame <- function(x, data = NULL, verbose = TRUE, ...) {
 #' @export
 get_predicted.lm <- function(x,
                              data = NULL,
-                             predict = "relation",
+                             predict = c("expectation", "link", "prediction", "relation"),
```

Comment:
In general, I'm personally not a big fan of putting the whole vector of possibilities in the definition. It's a very weird R-thing and it makes it less clear what the actual default is (especially for people that have some experience with any other languages that look at the list and be like *??what??* - it's really not intuitive or transparent that the default is actually the first element of the list). Yes one could argue that "it gives all the possibilities at first glance and no need to look at the arg description", which is especially true in the context for experienced R users - which are likely to be the users of insight

So we can keep it as is, especially since @strengejacke uses this form quite often too I think

Reply:
Yeah, but just because I didn't know about the `choices` argument in `match.arg()` ;-)

But indeed, if the list is too long, it might be better to mention the option in the description of the arguments. Not sure if it makes sense to have a consistent style here?

Reply:
I think so, too. In most cases, this "list" of options is not too long.